### PR TITLE
ADD BE UDF bitmap_and  and bitmap_or

### PR DESCRIPTION
**Pull request by DanyBin**
1. Add BE udf bitmap_and and bitmap_or
2.Test environment, modify production bitmap

### CODE REVIEW

File: be/src/exprs/bitmap_function.cpp [hunk 1]
```diff
@@ -388,6 +388,23 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
     return result;
 }
 
+StringVal BitmapFunctions::bitmap_or(FunctionContext* ctx, const StringVal& src, const StringVal& dst){
```
Comment (imay) [inline be/src/exprs/bitmap_function.cpp]: src and dst can be null, you should handle this case.

File: be/src/exprs/bitmap_function.cpp [hunk 2]
```diff
@@ -388,6 +388,23 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
...
+    RoaringBitmap src_bitmap ((char*)src.ptr);
```
Comment (imay) [inline be/src/exprs/bitmap_function.cpp]: src and dst can be already deserialized, you should handle the different cases.

File: be/test/exprs/bitmap_function_test.cpp [hunk 3]
```diff
@@ -29,18 +29,16 @@
 namespace doris {
 
 StringVal convert_bitmap_to_string(FunctionContext* ctx, RoaringBitmap& bitmap) {
```
Comment (imay) [inline be/test/exprs/bitmap_function_test.cpp]:
```suggestion
StringVal convert_bitmap_to_string(FunctionContext* ctx, const RoaringBitmap& bitmap) {
```

**Review by imay — CHANGES_REQUESTED** [review]
should update documents for new added functions

File: be/src/exprs/bitmap_function.cpp (hunk 1 above)
Comment (DanyBin) [reply be/src/exprs/bitmap_function.cpp]: OK. I will update doc and fix this case.

File: be/src/exprs/bitmap_function.cpp [hunk 4]
```diff
@@ -388,6 +388,51 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
...
+        } else{
+            bitmap.merge(RoaringBitmap ((char*)dst.ptr));
+        }
+    }
+
+    StringVal result(ctx,bitmap.size());
+    bitmap.serialize((char*)result.ptr);
+    return result;
+}
+StringVal BitmapFunctions::bitmap_and(FunctionContext* ctx, const StringVal& src, const StringVal& dst){
```
Comment (blueChild) [inline be/src/exprs/bitmap_function.cpp]: is the third parameter called ‘dst’ appropriate?

**Pull request closed without merging**